### ISSUE TITLE
Search: search accuracy on page title

### ISSUE THREAD

**Issue by Blackbaud-BrandonJones** [blackbaud/stache2#510]
### Expected behavior
I should be able to search on page title and find the page I am looking for.

### Actual behavior
Sometime the page doesn't show up accurately at all.  

### Steps to reproduce
https://host.nxt.blackbaud.com/engsys/generate-1bb-access-token
copy the page title `'Generate 1BB Access Token for Auth User' ` and search.  

Isaac Aggrey [5 minutes ago]
How about an easy way to provide feedback? @brandon.jones I was about to say the search was decent until I could copy the heading of a page and it didn't show up in the search.

Ex. https://host.nxt.blackbaud.com/engsys/generate-1bb-access-token - search for 'Generate 1BB Access Token for Auth User' with the docs search and it doesn't show up. The closest you get is the https://host.nxt.blackbaud.com/engsys/tools about 8 links down in the search.

**Comment by Blackbaud-TimPepper** [comment]
Mentioning @Blackbaud-IsaacAggrey as the initial reporter for this issue. 

To resolve this issue we will need to add Stache tags to the Generate 1BB Access Token for Auth User site, which will allow it to be indexed in Search. We should also do research to figure out if any other sites should be included in the effort. 

We will need to create a component to allow us to omit content from the web scraping that captures the site text, for security reasons.

**Comment by Blackbaud-LindseyRix** [comment]
We have made improvements to how Search works and believe this issue is no longer happening. If you are noticing issues with your page titles showing up in search results, please re-open and add a comment.

**Comment by Blackbaud-IsaacAggrey** [comment]
@Blackbaud-LindseyRix I still see this behavior at docs.blackbaud.com - https://docs.blackbaud.com/docs-search/search/Generate%201BB%20Access%20Token%20for%20Auth%20User and searching on the ES docs: https://host.nxt.blackbaud.com/engineering-system-docs/

**Comment by Blackbaud-BrandonJones** [comment]
@Blackbaud-IsaacAggrey That's correct, the page you're linking does not actually use Stache.  The Engineering System itself uses Stache for their documentation, the EngSys SPA does not use Stache, so it's contents are not searchable.

**Comment by Blackbaud-IsaacAggrey** [comment]
Thanks @Blackbaud-BrandonJones - could there be a feature request to index it and provide it in the search? :)

**Comment by Blackbaud-TimPepper** [comment]
I believe we previously talked about creating a small effort to track down other pages like this that people would find useful in search. We should create a small task to compile those and get them all in search at once, if we can.

**Comment by Blackbaud-LindseyRix** [comment]
I'll put this item in the list of things to discuss in our next backlog grooming, then we'll get back with you @Blackbaud-IsaacAggrey!
